<?xml version='1.0' encoding='utf-8'?>

<launch>
  <node pkg="xmate_robot_test" type="place_bjj_demo" name="xmate_robot" output="screen" >
    <!--设置机械臂运行速度-->
    <param name="move_sped" value="0.3"/>
    <param name="monitor_state" value="false"/>
    
    <!--螺丝放置位置对应Ar码号-->
    <param name="LS_ID" value="2"/>
    <!--螺母放置位置对应Ar码号-->
    <param name="LM_ID" value="5"/>
    <!--钣金件放置位置对应Ar码号-->
    <param name="BJJ_ID" value="2"/>

    <!-- 钣金件从物料盘上面抓取参数 -->
    <!-- 钣金件从物料盘上面抓取时机械臂的x坐标偏移量(笛卡尔坐标系)，增大往后(默认：0.0) -->
    <param name="zpt_bjj_grasp_x" value="0.0430802"/>　<!-- 单位：m-->
    <!-- 钣金件从物料盘上面抓取时机械臂的y坐标偏移量(笛卡尔坐标系)，增大向左(默认：0.1115) -->
    <param name="zpt_bjj_grasp_y" value="-0.109"/>　<!-- 单位：m-->
    <!-- 钣金件从物料盘上面抓取时机械臂的z坐标偏移量(笛卡尔坐标系)，增大往下(默认：0.041234663) -->
    <param name="zpt_bjj_grasp_z" value="0.0420"/>　<!-- 单位：m-->
    
     <!-- 钣金件放到装配台上面放置参数 -->
    <!-- 钣金件放到装配台上面放置时机械臂的x坐标偏移量(笛卡尔坐标系)，增大往左(默认：0.0) -->
    <param name="zpt_bjj_place_x" value="0.018"/>　<!-- 单位：m-->
    <!-- 钣金件放到装配台上面放置时机械臂的y坐标偏移量(笛卡尔坐标系)，增大向前(默认：0.1115) -->
    <param name="zpt_bjj_place_y" value="0.056"/>　<!-- 单位：m 0.0945-->
    <!-- 钣金件放到装配台上面放置时机械臂的z坐标偏移量(笛卡尔坐标系)，增大往上(默认：0.041234663) -->
    <param name="zpt_bjj_place_z" value="0.216"/>　<!-- 单位：m-->
  </node>

</launch>
